docs/docs/configuration/overview.md

```diff
@@ -3,11 +3,26 @@ id: overview
 title: Overview
 ---
 
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
 `oauth2-proxy` can be configured via [config file](#config-file), [command line options](#command-line-options) or [environment variables](#environment-variables).
 
 ### Generating a Cookie Secret
 
-To generate a strong cookie secret use `python -c 'import os,base64; print(base64.urlsafe_b64encode(os.urandom(16)).decode())'`
+To generate a strong cookie secret use one of the below commands:
+
+<Tabs
+  defaultValue="python"
+  values={[
+    {label: 'Python', value: 'python'},
+    {label: 'Bash', value: 'bash'},
+    {label: 'OpenSSL', value: 'openssl'},
+  ]}>
+  <TabItem value="python"><code>python -c 'import os,base64; print(base64.urlsafe_b64encode(os.urandom(32)).decode())'</code></TabItem>
```

Comment:
Nice! Good call bumping this to 32 👍

Reply:
A hack I've also done in the past for Terraform is this:

```
# Valid 32 Byte Base64 URL encoding set that will decode to 24 []byte AES-192 secret
resource "random_password" "cookie_secret" {
  length           = 32
  override_special = "-_"
}
```
32 characters that look like Base64 will opportunistically decode to 24 bytes, without having to worry about padding on the end.

Reply:
Ooh a terraform example would be cool, I'll add that